test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -63,11 +63,11 @@ Describe "Get-Content" -Tags "CI" {
 	  Get-Content -Path $testPath2 -Last 1 | Should Be $fifthline
   }
   It "Should be able to get content within a different drive" {
-	  pushd env:
+	  Push-Location env:
 	  $expectedoutput = [Environment]::GetEnvironmentVariable("PATH");
     { Get-Content PATH } | Should Not Throw
 	  Get-Content PATH     | Should Be $expectedoutput
-  	popd
+  	Pop-Location
   }
   #[BugId(BugDatabase.WindowsOutOfBandReleases, 906022)]
   It "should throw 'PSNotSupportedException' when you set-content to an unsupported provider" -Skip:($IsLinux -Or $IsOSX) {
```

Comment:
Again, I can't speak to these tests, but perhaps `BugId(BugDatabase.WindowsOutOfBandReleases, 906022)` holds the key? 

Separately, if you have any pointers for why the PR doesn't work on _Windows_ (have yet to dig into it), that would be helpful.